### PR TITLE
146-user about field

### PR DESCRIPTION
**Pull request by ervinj1998**
agregando about a la base de datos

### CODE REVIEW

**Comment by Hawaiideveloper** [comment]
Screen Shot 2020-10-14 at 10 27 24 AM

**Review by Hawaiideveloper — CHANGES_REQUESTED** [review]
please leave comments as to what this does @ervinj1998

File: routes/user.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ const {
     allUsers, 
     getUser, 
     updateUser, 
-    deleteUser } = require("../controllers/user");  //getUser imported by object destruct
-
-
```
Comment (manzanarez83) [inline routes/user.js:9]: with the following variable:
**userPhoto,
 } = require("../controllers/user");**  
allows to see the photo on the Frontend.

File: routes/user.js [hunk 2]
```diff
@@ -4,9 +4,9 @@ const {
...
+    deleteUser,
+    userPhoto,
+ } = require("../controllers/user");  //getUser imported by object destruct
```
Comment (manzanarez83) [inline routes/user.js:9]: with the following variable:
**userPhoto,
 } = require("../controllers/user");**  
allows to see the photo on the Frontend.

**Pull request merged**